android-client-sdk/src/main/java/com/devcycle/sdk/android/api/DVCClient.kt

```diff
@@ -320,6 +320,31 @@ class DVCClient private constructor(
         }
     }
 
+    private fun refetchConfig(callback: DVCCallback<Map<String, Variable<Any>>>? = null) {
+        if (isExecuting.get()) {
+            configRequestQueue.add(UserAndCallback(this@DVCClient.user, callback))
+            Timber.d("Queued refetchConfig request")
+            return
+        }
+
+        isExecuting.set(true)
+        coroutineScope.launch {
+            withContext(coroutineContext) {
+                try {
+                    fetchConfig(this@DVCClient.user)
```

Comment:
what happens when there's an identify already in flight? will this be the correct user that we need to fetch?

Reply:
hmm yeah it would take the user from the most recent successful config request, not any requests in progress. i was trying to figure out a way to check the most recent user already in the queue and use that, but i don't want to have to iterate through the whole queue to find it. So i ended up just adding a new variable to track latestIdentifiedUser at the beginning of every identify call and that seems to be working as expected